rasa/core/policies/memoization.py

```diff
@@ -195,8 +195,22 @@ def _recall_states(self, states: List[State]) -> Optional[Text]:
         return self.lookup.get(self._create_feature_key(states))
 
     def recall(
-        self, states: List[State], tracker: DialogueStateTracker, domain: Domain
+        self,
+        states: List[State],
+        tracker: DialogueStateTracker,
+        domain: Domain,
+        **kwargs: Any,
```

Comment:
Our out-of-the-box policies should unwrap default `kwargs` and make them explicit as soon as possible (which would be `policy.predict_action_probabilities`. This makes the code more explicit and also enables type checking.